courses/course_planning/specialcourse.md

```diff
@@ -0,0 +1,63 @@
+---
+id: specialcourse
+title: Special Courses
+sidebar_label: Special Courses
+slug: /course_planning/specialcourse
+---
+
+_Last updated on 2022/07/29_
+
+In addition to the classes listed in the [course offering table](/wiki/courses/), there are other CS-related classes that may not have a particular term of offering but could still be of your interest.
+
+## Selected Topics
+
+These classes offers different topics from year to year. For higher year courses, they may also offer lecture components from other graduate courses.
+When the topic changes, it may be repeated for credit. However, you should seek a CS advisor before you take these courses.
+
+-   COMP-2750 (Prerequisite: COMP-1000 or MATH-1720, and COMP-1410)
+-   COMP-3750 (Prerequisite: COMP-2120, COMP-2540, and COMP-2560)
+-   COMP-3760 (Prerequisite: COMP-2120, COMP-2540, and COMP-2560)
+-   COMP-4750 (Prerequisite: to be in Semester 7 and 8 and obtained instructor's permission)
+-   COMP-4760 (Prerequisite: to be in Semester 7 and 8 and obtained instructor's permission)
+
+## Co-op Work Terms
+
+These classes are exclusive to students who enrolled in Co-op programs. They are Pass/nonPass classes that requires completion of 420+ hours,
+and students must pass these courses to continue in Co-op.
+
+-   COMP-2980: Co-op Work Term I
+-   COMP-3980. Co-op Work Term II
+-   COMP-4970. Co-op Work Term III
+-   COMP-4980. Co-op Work Term IV
+
+## Project Courses
+
+In addition to the courses listed in the table above, COMP-4700 is a course where students can complete a project using computing tools of their choice.
```

Comment:
I may be wrong but I dont know a `COMP-4700` course

Reply:
I found this course in the academic calendar and decided to put it here because the course offerings pdf made it look like it wasn't offered at all 🤣 
It was offered in Summer 2022 so I think UWindsor will still offer it in the future

![image](https://user-images.githubusercontent.com/55353492/184509397-a3b1649b-a1de-435d-89c0-448698c54635.png)

Reply:
Ahh, I havent seen this course before. Sounds good then